Tools/PIDML/kaon_pid.cxx

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// This task produces invariant mass vs. momentum and dEdX in TPC vs. momentum
+/// for Kaons using ML PID from the PID ML ONNX Model.
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/PIDResponse.h"
+#include "CommonConstants/PhysicsConstants.h"
+#include "TMath.h"
+#include <math.h>
+#include "TLorentzVector.h"
+#include "TDatabasePDG.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Tools/PIDML/pidOnnxModel.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+namespace o2::aod
+{
+using MyCollisions = soa::Join<aod::Collisions,
+                               aod::EvSels,
+                               aod::Mults>;
+using MyTracks = soa::Join<aod::FullTracks, aod::TracksExtra, aod::pidTOFbeta,
+                           aod::TOFSignal, aod::TracksDCA>;
+using MyCollision = MyCollisions::iterator;
+using MyTrack = MyTracks::iterator;
+} // namespace o2::aod
+
+struct kaon_pid {
```

Comment:
Please do name your task `KaonPidTask`, with Pascal case. It is very important to use Pascal case for task struct names as later this is converted into the runtime task name.